packages/init/src/utils/auth/helpers.ts

```diff
@@ -81,13 +81,17 @@ const authenticationHandler =
     };
   };
 
+function stripTrailingSlash(str: string) {
+  return str.endsWith("/") ? str.slice(0, -1) : str;
+}
+
 function buildServer(base: string, port: number, host: string): hapi.Server {
   const server = hapi.server({
     port,
     host,
     routes: {
       cors: {
-        origin: [base],
+        origin: [stripTrailingSlash(base)],
```

Comment:
more trailing slash issues

Reply:
Yeah but here the function is important as it can cause CORS issues while logging.